documentation/operators/src/nodes/troubleshooting.md

```diff
@@ -262,23 +262,6 @@ thread 'tokio-runtime-worker' panicked at 'Failed to create TCP listener: Os { c
 
 Then you need to `--announce-host <PUBLIC_IP>` and `--host <LOCAL_IP>` on startup. This issue is addressed [above](./troubleshooting.md#missing-`announce-host`-flag)
 
-<!--- Commenting for now:
```

Comment:
why is this commented out?

Reply:
This has been commenting since ages, I deleted it, we can return it if it's still actual.